Joplin (portable)

super-lurker opened the pull request.

FlotterCodername reviewed (approved):
Thanks! Looks nice.
I just updated the gist writing style and updated the Available Apps section.

I'm going to have a place to credit all contributors.
May I credit you? If so, how do you want to be credited?

super-lurker commented:
> Thanks! Looks nice. I just updated the gist writing style and updated the Available Apps section.
> 
> I'm going to have a place to credit all contributors. May I credit you? If so, how do you want to be credited?

sure, my username would do.
